samples/bluetooth/peripheral_rscs/Kconfig

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "Kconfig.zephyr"
+
+menu "Nordic Running Speed and Cadence Service sample"
+
+config BT_RSCS_SECURITY_ENABLED
+	bool "Enable security"
+	default y
+	select BT_SMP
+	select BT_SETTINGS
+	select FLASH
+	select FLASH_PAGE_LAYOUT
+	select FLASH_MAP
+	select NVS
+	select SETTINGS
+	help
+	  "Enable BLE security for the Running Speed and Cadence Service"
```

Comment:
Do not use BLE. Instead, use Bluetooth LE. See https://projecttools.nordicsemi.no/confluence/display/TECHDOC/Terms%2C+abbreviations%2C+and+spelling+rules#Terms,abbreviations,andspellingrules-B